core/pom.xml

```diff
@@ -216,5 +216,11 @@ Import-Package: javax.annotation;version=0.0.0,*
             <artifactId>commons-csv</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.22</version>
+            <scope>compile</scope>
```

Comment:
This is a great idea to add _lombok_! But please add it to the `dependencyManagement` section of the main POM file and then copy here without version . This just works for any sort of Maven project.
Also, compile scope seems to be redundant. You don't want lombok to be bundled and deployed to the server

Reply:
done